repolint.json

```diff
@@ -169,6 +170,58 @@
         }
       }
     },
+    "source-license-headers-exist": {
+      "level": "error",
+      "rule": {
+        "type": "file-starts-with",
+        "options": {
+          "globsAll": [
+            "**/*.py",
+            "**/*.js",
+            "**/*.c",
+            "**/*.cc",
+            "**/*.cpp",
+            "**/*.h",
+            "**/*.ts",
+            "**/*.sh",
+            "**/*.rs",
+            "**/*.java",
+            "**/*.go",
+            "**/*.bbclass"
+          ],
+          "skip-paths-matching": {
+            "patterns": [
+              "babel.config.js",
+              "build\/",
+              "jest.config.js",
+              "node_modules\/",
+              "types\/"
+            ]
+          },
+          "lineCount": 40,
+          "patterns": [
+            "Copyright.*Qualcomm Innovation Center, Inc|©.*Qualcomm Innovation Center, Inc",
+            "SPDX-License-Identifier"
+          ],
+          "flags": "i"
+        }
+      }
+    },
```

Comment:
Looks like you copy pasta the whole previous "source-license-headers-exist" block?

Reply:
ha ha ha! Whoops!